os/net/netdev/netdev_ioctl.c

```diff
@@ -1003,6 +1006,58 @@ static int netdev_rtioctl(FAR struct socket *sock, int cmd, FAR struct rtentry *
 }
 #endif							/* CONFIG_NET_ROUTE */
 
+#if CONFIG_NET_LWIP
+/****************************************************************************
+ * Function: lwip_func_ioctl
+ *
+ * Description:
+ *   Call lwip API
+ *
+ * Parameters:
+ *   arg Type of the information to get
+ *
+ * Returned Value:
+ *   0 on success, an error code on failure
+ *
+ ****************************************************************************/
+
+int lwip_func_ioctl(void *arg)
+{
+	int ret = EAI_FAIL;
+	struct lwip_info *in_arg = (struct lwip_info *)arg;
+	if (!in_arg) {
+		return ret;
+	}
+	const char *hostname = NULL;
+	const char *servname = NULL;
+	const struct addrinfo *hints = NULL;
+	struct addrinfo *res = NULL;
+	struct addrinfo *ai = NULL;
+
+	switch (in_arg->type) {
+	case GETADDRINFO:
+		hostname = in_arg->host_name;
+		servname = in_arg->serv_name;
+		hints = in_arg->ai_hint;
+		ret = lwip_getaddrinfo(hostname, servname, hints, &res);
```

Comment:
I think below is more comfortable option, because lwip_getarddrinfo returns each of diffrent result.
if (ret < 0) {
    error_debug("blahblah %d\n", errno);
    return something;
}
if (res) {
    in_arg->ai_res = res;
}
```

Reply:
I think so I will fix.